pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/PicklistQuestionDao.java

```diff
@@ -178,14 +178,16 @@ default void insertNestedOptions(long questionId, PicklistOptionDef option,
 
     /**
      * Create a new picklist option for given question.
+     * 
+     * <p>The `option` parameter <b>will be modified</b> by this method.
      *
      * @param questionId   the associated picklist question
      * @param option       the picklist option definition
      * @param displayOrder the display order number for the option
      * @param revisionId   the revision to use, will be shared for all created data
      * @return the option id
      */
-    default long insertOption(long questionId, PicklistOptionDef option, int displayOrder, long revisionId) {
+    public default long insertOption(long questionId, PicklistOptionDef option, int displayOrder, long revisionId) {
```

Comment:
Same here. Is there any reason why you want to explicitly define the method as public?

Reply:
In general, I feel explicit is better than implicit when it comes to visibility, but in these cases they had been represented in my mind more as classes than interfaces, and got assigned `package-private` as the default visibility and were mechanically updated when they were needed outside their parent scope & package.